code/_onclick/click.dm

```diff
@@ -415,12 +412,12 @@
 	A.AltClick(src)
 
 /atom/proc/AltClick(mob/user)
+	var/turf/T = get_turf(src)
 	if(!can_interact(user))
 		return FALSE
 	if(SEND_SIGNAL(src, COMSIG_CLICK_ALT, user) & COMPONENT_CANCEL_CLICK_ALT)
 		return
-	var/turf/T = get_turf(src)
-	if(T && (isturf(loc) || isturf(src)) && user.TurfAdjacent(T))
+	else if((T && (isturf(loc)) || (isturf(src)) && user.TurfAdjacent(T)) && !HAS_TRAIT(user, TRAIT_MOVE_VENTCRAWLING))
```

Comment:
why else? also why does each check have () wrapped around it? shouldn't it only need to be where it was before? since 
`(thing && other thing)` is the same as `((thing) && other thing)`

Reply:
else is leftover from my trying to figure out how to prevent the statpanel turf tab from being created when you alt click to interact with something like a chair or toggling power to a cryo bed. <- will remove the else

the parenthesis are just because not having them doesn't look as neat to me but it looks like i did an oopsie with them

Reply:
both points addressed by https://github.com/tgstation/tgstation/pull/68965/commits/62816effdaf1ecdd964c0bb5eb15cd5d67251988